include/libtorrent/piece_block.hpp

```diff
@@ -48,7 +48,7 @@ namespace libtorrent
 		{
 		}
 		piece_index_t piece_index;
-		int block_index;
+		int block_index = 0;
```

Comment:
these were kind of deliberately left uninitialized. but, if this field should be initialized, surely ``piece_index`` should too. ``piece_index{0}``